packages/ui/src/components/Question/Question.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { AccordionButton, AccordionIcon, AccordionItem, AccordionPanel, Box } from '@chakra-ui/react';
+
+import { Typography } from '../Typography';
+
+export type QuestionProps = {
+  title: string;
+  content: string;
+};
+
+export const Question = ({ title, content }: QuestionProps) => {
+  return (
+    <AccordionItem borderColor="gray.200">
+      <h2>
+        <AccordionButton>
+          <Box flex="1" textAlign="left">
```

Comment:
❓ Do we need this Box here? Wouldn't it work when we pass those two props to the Typography component?

Reply:
It does work this way